src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/03 Options.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Options'
+excerpt: 'Options allows to configure the tracing instrumentation behavior.'
+---
+
+Use the `Options` object to configure the tracing instrumentation behavior. It is used during the instantiation of a [`Client`](/javascript-api/k6-experimental/tracing/client) instance and also as a parameter to the [`instrumentHTTP`](/javascript-api/k6-experimental/tracing/instrumenthttp) function. It controls the general behavior of the tracing instrumentation and is unspecific to any particular tracing client instance.
+
+## Options
+
+| Option name | Type | Default | Description |
+| :----------- | :------- | :------ | :------------------------------------------------------------------------------------------------------------------------------ |
+| `propagator` | `string` | `w3c` | The trace context propagation format. Currently supported: `w3c` and `jaeger`. |
+| `sampling` | `number` | `100` | The probability of each request having its `sampled` flag set to true. Its value should be within the `0 <= n <= 100` bounds. |
```

Comment:
Given that this isn't merged - shall we document it?

Reply:
Good catch, had added it preemptively. Dropping it 👍🏻